fdbserver/GrvProxyServer.actor.cpp

```diff
@@ -110,7 +112,13 @@ struct GrvProxyStats {
 	                          id,
 	                          SERVER_KNOBS->LATENCY_METRICS_LOGGING_INTERVAL,
 	                          SERVER_KNOBS->LATENCY_SAMPLE_SIZE),
-	    grvLatencyBands("GRVLatencyBands", id, SERVER_KNOBS->STORAGE_LOGGING_DELAY) {
+	    grvLatencyBands("GRVLatencyBands", id, SERVER_KNOBS->STORAGE_LOGGING_DELAY),
+	    grvConfirmEpochLiveDist(Histogram::getHistogram(LiteralStringRef("GrvProxy"),
+	                                                    LiteralStringRef("grvConfirmEpochLive"),
+	                                                    Histogram::Unit::microseconds)),
+	    grvRawDist(Histogram::getHistogram(LiteralStringRef("GrvProxy"),
+	                                       LiteralStringRef("grvRawRpc"),
```

Comment:
Maybe change `grvRawRpc` to `grvGetCommittedVersionRpc`

Reply:
Done